src/faucet/utils.rs

```diff
@@ -122,6 +122,11 @@ pub fn format_balance(balance: &TokenAmount, unit: &str) -> String {
     )
 }
 
+/// Constructs a URL to lookup the faucet history for a given address.
+pub fn format_address_url(base_url: &str, address: &str) -> String {
+    format!("{}address/{}", base_url, address)
+}
+
```

Comment:
I'd rather we use strongly typed `Url` and not raw strings when dealing with URLs. This could catch potential issues in the future.

Reply:
fixed